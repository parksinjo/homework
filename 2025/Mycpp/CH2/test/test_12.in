9
HongGilDong 0 71.5 1 :Gwangju Nam-gu Bongseon-dong 21:
8
0
