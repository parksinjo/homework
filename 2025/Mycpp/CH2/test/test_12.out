
+++++++++++++++++++++ Person Structure Test Menu +++++++++++++++++++++
+ 0.Exit 1.printlnTest 2.initTest 3.getter 4.setter 5.isSameTest     +
+ 6.whatAreYouDoingTest 7.NoBoolAlphaOutput 8.BoolAlphaOutput        +
+ 9.NoBoolAlphaInputPerson 10.BoolAlphaInputPerson 11.DataTypeSize() +
++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++
Menu item number? 9
input person information:
HongGilDong 0 71.5 1 :Gwangju Nam-gu Bongseon-dong 21:
HongGilDong 0 71.5 1 :Gwangju Nam-gu Bongseon-dong 21:

+++++++++++++++++++++ Person Structure Test Menu +++++++++++++++++++++
+ 0.Exit 1.printlnTest 2.initTest 3.getter 4.setter 5.isSameTest     +
+ 6.whatAreYouDoingTest 7.NoBoolAlphaOutput 8.BoolAlphaOutput        +
+ 9.NoBoolAlphaInputPerson 10.BoolAlphaInputPerson 11.DataTypeSize() +
++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++
Menu item number? 8
 0 0 false ::
LeeSoonShin 0 70.1 true :Gangnam Seoul:
Hong,gildong 1 60.2 false :Jongno-gu, Busan:
HongGilDong 0 71.5 true :Gwangju Nam-gu Bongseon-dong 21:
init(u1 10 60 false :Jongno-gu, Seoul:)
init(u2 0 71.5 true :Gwangju Nam-gu Bongseon-dong 21:)
HongGilDong 0 71.5 true :Gwangju Nam-gu Bongseon-dong 21:
name:HongGilDong, id:0, weight:71.5, married:true, address:Gwangju Nam-gu Bongseon-dong 21
u 0 71.5 true :Gwangju Nam-gu Bongseon-dong 21:
HongGilDong 0 71.5 true :Gwangju Nam-gu Bongseon-dong 21:
p.isSame(): false
HongGilDong is taking a rest.

+++++++++++++++++++++ Person Structure Test Menu +++++++++++++++++++++
+ 0.Exit 1.printlnTest 2.initTest 3.getter 4.setter 5.isSameTest     +
+ 6.whatAreYouDoingTest 7.NoBoolAlphaOutput 8.BoolAlphaOutput        +
+ 9.NoBoolAlphaInputPerson 10.BoolAlphaInputPerson 11.DataTypeSize() +
++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++
Menu item number? 0
Good bye!!
